cm-gateway/common.tfvars

```diff
@@ -32,3 +22,20 @@ issuer_name = "gateway-issuer"
 
 # Hostname that demo gateway to use. Default is "example.com"
 gateway_hostname = "example.com"
+
+# Name of testing gateway. Default is "cmsolver"
+gateway_name = "cmsolver"
+
+
+# The following parameters must be used in combination
+# 1. issuer_type is "SelfSigned" and envoy.service.type is "NodePort", which can be tested in a local kind environment.
+# 2. issuer_type is "ACME" and envoy.service.type is "LoadBalancer", which can be tested in a cloud environment such as GKE.
```

Comment:
Now it is only tested, if there will not problem, I will do other works, like update readme.